bench/wtperf/wtperf.c

```diff
@@ -559,8 +559,6 @@ worker(void *arg)
 					trk = &thread->truncate;
 				else
 					trk = &thread->truncate_sleep;
-				/* Pause between truncate attempts */
-				(void)usleep(1000);
```

Comment:
Was this sleep in here for a different workload, where truncate easily keeps up? i.e: Will taking this out cause the truncate thread to busy-wait in some configurations?